.github/workflows/source_bitbucket.yml

```diff
@@ -0,0 +1,48 @@
+name: Source Plugin Bitbucket Workflow
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    paths:
+      - "plugins/source/bitbucket/**"
+      - ".github/workflows/source_bitbucket.yml"
+  push:
+    branches:
+      - main
+    paths:
+      - "plugins/source/bitbucket/**"
+      - ".github/workflows/source_bitbucket.yml"
+
+jobs:
+  plugins-source-bitbucket:
+    timeout-minutes: 30
+    name: "plugins/source/bitbucket"
+    runs-on: large-ubuntu-monorepo
+    defaults:
+      run:
+        working-directory: ./plugins/source/bitbucket
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "temurin"
+          java-version: "18"
+          cache: "gradle"
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@56b90f209b02bf6d1deae490e9ef18b21a389cd4
+      - name: Build package
+        run: ./gradlew build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v3
+        if: success() || failure() # always run even if the previous step fails
+        with:
+          report_paths: "**/build/test-results/test/TEST-*.xml"
+          require_passed_tests: true
+          fail_on_failure: true
```

Comment:
See https://github.com/cloudquery/cloudquery/blob/9dfb9aceef98fb3392b3274e4ffd22a9f3b7748e/.github/workflows/source_airtable.yml#L45 for a validate release job. You'll need to setup the Dockerfile first